src/components/cl/hier/allreduce/allreduce_rab.c

```diff
@@ -54,6 +54,11 @@ UCC_CL_HIER_PROFILE_FUNC(ucc_status_t, ucc_cl_hier_allreduce_rab_init,
         goto out;
     }
 
+    if (!SBGP_ENABLED(cl_team, NODE) && !SBGP_ENABLED(cl_team, NODE_LEADERS)) {
```

Comment:
this cannot happen - it is rather ucc_assert. Either of those SBGP must be enabled.

Reply:
I know, it's just to suppress warning. ucc_assert will not work because it's no op for release build

Reply:
can we do smth with NOLINT tricks ?

Reply:
updated, is it better now?